components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx

```diff
@@ -164,7 +162,10 @@ export const contactSalesCard = (
                 }
             </div>
             <div className='PrivateCloudCard__image'>
-                <img src={isFreeTrial ? freeTrialPrivateCloudImage : privateCloudImage}/>
```

Comment:
Here we had two images, we may be missing the one for the case of free trial?

Reply:
I had understood that we would use the same image for both scenarios. Maybe @matthewbirtch can help us to confirm this.

Reply:
<img width="981" alt="Captura de pantalla 2021-08-11 a las 22 47 04" src="https://user-images.githubusercontent.com/10082627/129101185-8cf3c842-7ed1-4fd9-8354-d2db128c39b2.png">

added the image and the condition back! @matthewbirtch @marianunez